recipes/ordpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ordpy" %}
+{% set version = "1.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ordpy-{{ version }}.tar.gz
+  sha256: 45a10bcad068854a4e0f76144d0d395c0fd702ea2ea3dbb469e7501be95ec7ee
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
```

Comment:
It looks like `numpy` is a `run` requirement`. This information is missing from the upstream `setup.py`. It's in the `pyproject.toml`, but as a build system requirement.